TRENZ.Lib.RazorMail.MailKit/Extensions/MailAddressExtensions.cs

```diff
@@ -2,10 +2,10 @@
 
 using TRENZ.Lib.RazorMail.Models;
 
-namespace TRENZ.Lib.RazorMail.MailKitExtensions;
+namespace TRENZ.Lib.RazorMail.Extensions;
```

Comment:
hmm, I had deliberately named it that because of overlapping / too-generic type names like MailboxAddress. This clarifies that those extensions are specifically for that particular implementation

Reply:
Any reason why the mailkit and systemnet implementations don't have their own namespaces?

Reply:
Is your question "…why the… implementations _should_ have their own namespaces"?

Reply:
No, the opposite. Why didn't they have their own namespaces already? Any reason? Otherwise I'd just add them

Reply:
If the `BaseSmtpMailClient` implementations come with their own namespace, it would be `TRENZ.Lib.RazorMail.MailKit.MailKitMailClient`, which seems a bit verbose to me?

Reply:
Yeah, but you virtually never write out the full name. IMHO it's better to be verbose than to be imprecise.

Reply:
> it's better to be verbose than to be imprecise

agreed